### PR TITLE
Copypasta Command

### PR DESCRIPTION
**Pull request by bolsomito**
---
name: copypasta
about: Fetches a random copypasta

---

**Command description**
Fetch a random copypasta post from /r/copypasta

**Usage**
`s!copypasta`

### CODE REVIEW

**Comment by LogGame-HU-DEaN** [comment]
I reckon the class should be renamed from Hmmm.

**Comment by LogGame-HU-DEaN** [comment]
Nvm, couple of seconds late.

**Comment by bolsomito** [comment]
Kappa

**Comment by bolsomito** [comment]
I do believe it is ready for reviewing.
@SwitchbladeBot/backend

File: src/commands/memes/copypasta.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    this.name = 'copypasta'
+    this.aliases = []
+  }
+
+  async run (message) {
+    const embed = new SwitchbladeEmbed(message.author)
+    const { title, selftext, permalink } = await Reddit.getRandomPostFromSubreddit('/r/copypasta')
+    let pasta
+    if (selftext.length > 2048) pasta = selftext.substr(0, 2045) + '...'
+    else pasta = selftext
```
Comment (davipatury) [inline src/commands/memes/copypasta.js]: `const pasta = selftext.length > 2048 ? selftext.substr(0, 2045) + '...' : selftext`

File: src/commands/memes/copypasta.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    message.channel.startTyping()
```
Comment (davipatury) [inline src/commands/memes/copypasta.js]: `startTyping` goes before `Reddit.getRandomPostFromSubreddit`

File: src/commands/memes/copypasta.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+const { Command, Reddit, SwitchbladeEmbed, Constants } = require('../../')
+
+module.exports = class Copypasta extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'copypasta'
+    this.aliases = []
```
Comment (davipatury) [inline src/commands/memes/copypasta.js]: Remove this line

**Pull request merged**